fraud-detection-new-isoscape/main.py

```diff
@@ -6,91 +6,58 @@
 from os import path
 import ee
 from dateutil import parser
-from origin_validation import ttest
 from datetime import datetime, timedelta
+from google.auth import compute_engine
 
 # Get function environment variable for GCP project ID to use for accessing Earth Engine.
 GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "river-sky-386919")
 
 # The path to the isoscapes folder in Earth Engine.
 ISOSCAPES_EE_PATH = f'projects/{GCP_PROJECT_ID}/assets/isoscapes'
 
-_VALIDATION_PASSED_LABEL = "Possible"
-_VALIDATION_FAILED_LABEL = "Not Likely"
-
 app = initialize_app()
 root = path.dirname(path.abspath(__file__))
 
 @functions_framework.http
 def reevaluate(request):
     # Initialize Earth Engine.
-    if os.path.isfile('key.json'):
-
-        # connection to the service account
-        service_account = 'earth-engine-sa@river-sky-386919.iam.gserviceaccount.com'
-        credentials = ee.ServiceAccountCredentials(
-            service_account, 'key.json')
-        ee.Initialize(credentials)
-        print('initialized with key')
-
-    # if in local env use the local user credential
-    else:
-        ee.Initialize()
+    # Authenticate to Earth Engine using service account creds
+    credentials = compute_engine.Credentials(scopes=['https://www.googleapis.com/auth/earthengine'])
```

Comment:
do we need to set up authorization/perms in the PRD project?  do we need to grant EE perms to the trigger service account?

Reply:
already done